### ISSUE TITLE
Cannot delete server if it has linked databases and their MySQL instance is unreachable

### ISSUE THREAD

**Issue by luizsusin** [pterodactyl/panel#2085]
**Background:**
* Panel or Daemon: Panel 
* Version of Panel/Daemon: 0.7.17 
* Server's OS: Ubuntu 18.04
* Your Computer's OS & Browser: Windows 10 - Google Chrome 83.0.4103.61

**Describe the bug**
I've migrated the files of an old server to a new one in a new node and completely wiped the machine that the old daemon was in. After a few hours, I've tried to delete the old server (with its node already unreachable) but it gives me an error. If I'm safely deleting, it gives me HTTP/E_CONN_REFUSED, which does make sense. However, if I'm forcibly deleting the server, it returns me an error 500.

PHP Version: 7.2.29-1 (built March 20 2020)

**To Reproduce**
1. Create a reachable node;
2. Create a server inside it;
3. Make the node unreachable;
4. Try to delete the server created (both safely and forcebly). 

**Expected behavior**
Being able to forcibly delete a server when the node is unreachable, so that you can also delete the unrechable node.

**Comment by parkervcp** [comment]
Is force delete not working for you? Can you get logs for this please?

`tail -n 100 /var/www/pterodactyl/storage/logs/laravel-$(date +%F).log | nc bin.ptdl.co 99`

**Comment by luizsusin** [comment]
Sure thing!
I actually got this tail a little bit longer (170) so that it would also take the warning that shows that it can't resolve the FQDN, which is expected since I've deleted it. However, it shows an error (Connection Refused) when trying to delete a database that was creted in this server.

Here it is:
https://bin.ptdl.co/7w3fn/

**Comment by DaneEveritt** [comment]
```
[2020-06-01 16:46:52] production.ERROR: QueryException: SQLSTATE[HY000] [2002] Connection refused (SQL: DROP DATABASE IF EXISTS `s1_authme`) in /home/ubuntu/pterodactyl/vendor/laravel/framework/src/Illuminate/Database/Connection.php:664
Stack trace:
#0 /home/ubuntu/pterodactyl/vendor/laravel/framework/src/Illuminate/Database/Connection.php(624): Illuminate\Database\Connection->runQueryCallback
#1 /home/ubuntu/pterodactyl/vendor/laravel/framework/src/Illuminate/Database/Connection.php(459): Illuminate\Database\Connection->run
#2 /home/ubuntu/pterodactyl/app/Repositories/Eloquent/DatabaseRepository.php(204): Illuminate\Database\Connection->statement
#3 /home/ubuntu/pterodactyl/app/Repositories/Eloquent/DatabaseRepository.php(181): Pterodactyl\Repositories\Eloquent\DatabaseRepository->run
#4 /home/ubuntu/pterodactyl/app/Services/Databases/DatabaseManagementService.php(127): Pterodactyl\Repositories\Eloquent\DatabaseRepository->dropDatabase
#5 /home/ubuntu/pterodactyl/app/Services/Servers/ServerDeletionService.php(124): Pterodactyl\Services\Databases\DatabaseManagementService->delete
#6 /home/ubuntu/pterodactyl/vendor/laravel/framework/src/Illuminate/Support/Collection.php(418): Pterodactyl\Services\Servers\ServerDeletionService->Pterodactyl\Services\Servers\{closure}
#7 /home/ubuntu/pterodactyl/app/Services/Servers/ServerDeletionService.php(125): Illuminate\Support\Collection->each
#8 /home/ubuntu/pterodactyl/app/Http/Controllers/Admin/ServersController.php(538): Pterodactyl\Services\Servers\ServerDeletionService->handle
#9 (0): Pterodactyl\Http\Controllers\Admin\ServersController->delete
```

**Comment by DaneEveritt** [comment]
This won't get fixed in `0.7.X` but we can fix it in the `1.0` builds.

**Comment by luizsusin** [comment]
Not a problem! I can manage to remove it from the database manually. Just wanted to contribute.
